### PR TITLE
adds forceAliasCreations clarification to auth docs

### PR DESCRIPTION
**Pull request by helgabalashova**
_Issue #, if available:_

_Description of changes:_
adds `forceAliasCreation` clarification to js `confirmSignUp` docs

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by abdallahshaban557 — DISMISSED** [review]
LGTM!

File: src/fragments/lib/auth/js/emailpassword.mdx [hunk 1]
```diff
@@ -68,6 +68,24 @@ async function confirmSignUp() {
 }
 ```
 
+As part of the options parameter, you can specify `forceAliasCreation` boolean. By default
+set to `True`. If set to `False` and email or phone number used for sign up confirmation
```
Comment (jimblanc) [inline src/fragments/lib/auth/js/emailpassword.mdx:72]: `and the email`

File: src/fragments/lib/auth/js/emailpassword.mdx [hunk 2]
```diff
@@ -68,6 +68,24 @@ async function confirmSignUp() {
 }
 ```
 
+As part of the options parameter, you can specify `forceAliasCreation` boolean. By default
```
Comment (jimblanc) [inline src/fragments/lib/auth/js/emailpassword.mdx:71]: `specify the`

**Review by jimblanc — APPROVED** [review]

**Review by abdallahshaban557 — APPROVED** [review]
LGTM

**Pull request merged**